hack/test-end-to-end-docker.sh

```diff
@@ -15,9 +15,8 @@ echo "[INFO] Starting containerized end-to-end test"
 
 unset KUBECONFIG
 
-TMPDIR="${TMPDIR:-"/tmp"}"
-BASETMPDIR="${BASETMPDIR:-${TMPDIR}/openshift-e2e-containerized}"
 setup_env_vars
+setup_temp_dir_vars "test/end-to-end/containerized/"
```

Comment:
@rhatdan what are the considerations for writing under `/tmp` versus `/run`? These scripts are all developer tools, used in testing environments.